### ISSUE TITLE
add support for tags for aws_opsworks_custom_layer

### ISSUE THREAD

**Issue by 5c077yP** [terraform-providers/terraform-provider-aws#3546]
Hi there,

### Terraform Version

```
Terraform v0.11.3
+ provider.aws v1.10.0
```

### Affected Resource(s)

- aws_opsworks_custom_layer

### Terraform Configuration Files
```hcl
resource "aws_opsworks_custom_layer" "test" {
  tags {
    MyTag = "MyValue"
  }
}
```

### Expected Behavior

it should create tags on the corresponding opsworks (custom) layer

### Actual Behavior

tags are not yet supported

**Comment by robertomoutinho** [comment]
almost a year later.... no tags yet at custom layers

**Comment by bflad** [comment]
Support for tagging in the Opsworks Layers resources has been merged and will release with version 2.54.0 of the Terraform AWS Provider, later this week. Thanks to @DrFaust92 for the implementation. 👍
